ang/civicase/case/actions/services/email-case-action.service.js

```diff
@@ -188,12 +188,13 @@
             action: 'add',
             hideDraftButton: 1,
             reset: 1,
-            cid: contactIDs.join(',')
+            cid: contactIDs.join(','),
+            caseid: model.caseIds[0]
           }
         };
 
-        if (model.caseIds.length === 1) {
-          popupPath.query.caseid = model.caseIds[0];
+        if (model.caseIds.length > 1) {
+          popupPath.query.allCaseIds = model.caseIds.join(',');
```

Comment:
Can you ask an FE dev to quickly check the modification to this JS file, it could be possible that we need to add a test for this change also.